test/position-managers/PositionManager.modifyLiquidities.t.sol

```diff
@@ -906,60 +917,68 @@ contract PositionManagerModifyLiquiditiesTest is Test, PosmTestSetup, LiquidityF
+                        : tickLower > TickMath.getTickAtSqrtPrice(sqrtPriceX96)
+                )
+        ) {
             vm.expectRevert(Position.CannotUpdateEmptyPosition.selector);
-            lpm.modifyLiquidities(actions, _deadline);
+            lpm.modifyLiquidities(planner.encode(), _deadline);
         } else {
             // MINT FROM DELTAS.
-            lpm.modifyLiquidities(actions, _deadline);
+            lpm.modifyLiquidities(planner.encode(), _deadline);
 
-            uint256 balance0After = fotKey.currency0.balanceOf(address(this));
-            uint256 balance1After = fotKey.currency1.balanceOf(address(this));
-            uint256 balance0PMAfter = fotKey.currency0.balanceOf(address(manager));
-            uint256 balance1PMAfter = fotKey.currency1.balanceOf(address(manager));
+            balance0._after = fotKey.currency0.balanceOf(address(this));
+            balance1._after = fotKey.currency1.balanceOf(address(this));
+            balance0PM._after = fotKey.currency0.balanceOf(address(manager));
+            balance1PM._after = fotKey.currency1.balanceOf(address(manager));
 
             // Calculate the expected resulting balances used to create liquidity after the fee is applied.
-            uint256 amountInFOT = currency0IsFOT ? amount0 : amount1;
-            uint256 expectedFee = amountInFOT.calculatePortion(bips);
-            (uint256 expected0, uint256 expected1) = currency0IsFOT
-                ? (balance0 - balance0After - expectedFee, balance1 - balance1After)
-                : (balance0 - balance0After, balance1 - balance1After - expectedFee);
-
-            assertEq(expected0, balance0PMAfter - balance0PM);
-            assertEq(expected1, balance1PMAfter - balance1PM);
-
-            // the liquidity that was created is a diff of the balance change
-            uint128 expectedLiquidity = LiquidityAmounts.getLiquidityForAmounts(
-                sqrtPriceX96,
-                TickMath.getSqrtPriceAtTick(tickLower),
-                TickMath.getSqrtPriceAtTick(tickUpper),
-                expected0,
-                expected1
-            );
-
-            assertEq(lpm.ownerOf(tokenId), address(this));
-            assertEq(lpm.getPositionLiquidity(tokenId), expectedLiquidity);
-
+            Balance memory expected;
+            {
+                bool currency0IsFOT = fotKey.currency0 == Currency.wrap(address(fotToken));
+                uint256 expectedFee = (currency0IsFOT ? amount0 : amount1).calculatePortion(bips);
+                (expected._0, expected._1) = currency0IsFOT
+                    ? (balance0.before - balance0._after - expectedFee, balance1.before - balance1._after)
+                    : (balance0.before - balance0._after, balance1.before - balance1._after - expectedFee);
+            }
+            assertEq(expected._0, balance0PM._after - balance0PM.before);
+            assertEq(expected._1, balance1PM._after - balance1PM.before);
```

Comment:
i guess given `_0` `_1` and `_after` have to have `_`, I'd like `before` to be `_before`